components/molecule/dataCounter/src/index.js

```diff
@@ -67,28 +65,19 @@ const MoleculeDataCounter = ({
     }
   }
 
-  const removeDigit = e => {
-    if (internalValue.length) {
-      const newValue = internalValue.slice(0, -1)
-      setInternalValue(newValue)
-      onChange(e, {value: newValue})
-    }
-  }
-
-  const addDigit = (e, {key: digit}) => {
-    const newValue = internalValue + digit
-    if (newValue.length <= 2) {
-      setInternalValue(newValue)
-      onChange(e, {value: newValue})
+  const handleChange = (e, {value}) => {
+    const numericValue = parseInt(value, 10)
+    if (value.length <= 2 && !isNaN(numericValue)) {
+      const strNewValue = String(numericValue)
```

Comment:
This 3 lines are repeated 3 times in`incrementValue`, `decrementValue` and here `handleChange`. What do you think about wrap it in a function and reutilize this code?

Reply:
done!